frontend/workflows/k8s/src/delete-pod.tsx

```diff
@@ -34,14 +33,16 @@ const PodIdentifier: React.FC<ResolverChild> = ({ resolverType, notes = [] }) =>
   return <Resolver type={resolverType} searchLimit={1} onResolve={onResolve} notes={notes} />;
 };
 
-const PodDetails: React.FC<WizardChild> = () => {
+const PodDetails: React.FC<VerifyChild> = ({ notes = [] }) => {
   const { onSubmit, onBack } = useWizardContext();
   const resourceData = useDataLayout("resourceData");
   const instance = resourceData.displayValue() as IClutch.k8s.v1.Pod;
+  const locationNotes = notes.filter(note => note.location === "verify");
 
   return (
     <WizardStep error={resourceData.error} isLoading={resourceData.isLoading}>
       <strong>Pod Details</strong>
+      <NotePanel notes={locationNotes} />
```

Comment:
Should this be a conditional render if there are no notes?

Reply:
I think it'll just render a grid with no children so to the end user it's not a huge change.